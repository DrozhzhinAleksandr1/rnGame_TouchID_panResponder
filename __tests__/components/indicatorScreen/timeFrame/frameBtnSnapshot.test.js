import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FrameBtn } from '../../../../app/components/indicatorScreen/timeFrame/frameBtn';

const mockFnc = () =>{}

it('create snapshot FrameBtn', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FrameBtn isChosen={true} frameBtnText={"1 мин"} showBtn={mockFnc}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
