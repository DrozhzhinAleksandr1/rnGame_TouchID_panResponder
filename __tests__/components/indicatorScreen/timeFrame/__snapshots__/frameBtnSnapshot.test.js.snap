// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot FrameBtn 1`] = `
<TouchableWithoutFeedback
  onPress={[Function]}
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1D2E3E",
        "borderColor": "#FFFFFF",
        "borderWidth": 1,
        "height": 26,
        "justifyContent": "center",
        "marginHorizontal": 5,
        "width": 54,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "Roboto-Regular",
          "fontSize": 12,
          "lineHeight": 14,
        }
      }
    >
      1 мин
    </Text>
  </Component>
</TouchableWithoutFeedback>
`;
